opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchPhraseQueryTest.java

```diff
@@ -104,9 +106,175 @@ public void test_zero_terms_query_parameter_lower_case() {
+        () -> matchPhraseQuery.build(new MatchPhraseExpression(
+            arguments, MatchPhraseQueryTest.this.matchPhraseQueryName)));
+  }
+
+  @Test
+  public void test_SyntaxCheckException_when_one_argument_matchphrase_syntax() {
+    List<Expression> arguments = List.of(dsl.namedArgument("field", "test"));
+    assertThrows(SyntaxCheckException.class,
+        () -> matchPhraseQuery.build(new MatchPhraseExpression(
+            arguments, MatchPhraseQueryTest.this.matchPhraseQueryName)));
+
+  }
+
+  @Test
+  public void test_SyntaxCheckException_when_invalid_parameter_matchphrase_syntax() {
+    List<Expression> arguments = List.of(
+        dsl.namedArgument("field", "test"),
+        dsl.namedArgument("query", "test2"),
+        dsl.namedArgument("unsupported", "3"));
+    Assertions.assertThrows(SemanticCheckException.class,
+        () -> matchPhraseQuery.build(new MatchPhraseExpression(
+            arguments, MatchPhraseQueryTest.this.matchPhraseQueryName)));
+  }
+
+  @Test
+  public void test_analyzer_parameter_matchphrase_syntax() {
+    List<Expression> arguments = List.of(
+        dsl.namedArgument("field", "t1"),
+        dsl.namedArgument("query", "t2"),
+        dsl.namedArgument("analyzer", "standard")
+    );
+    Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(
+        arguments, MatchPhraseQueryTest.this.matchPhraseQueryName)));
+  }
+
+  @Test
+  public void build_succeeds_with_two_arguments_matchphrase_syntax() {
+    List<Expression> arguments = List.of(
+        dsl.namedArgument("field", "test"),
+        dsl.namedArgument("query", "test2"));
+    Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(
+        arguments, MatchPhraseQueryTest.this.matchPhraseQueryName)));
+  }
+
+  @Test
+  public void test_slop_parameter_matchphrase_syntax() {
+    List<Expression> arguments = List.of(
+        dsl.namedArgument("field", "t1"),
+        dsl.namedArgument("query", "t2"),
+        dsl.namedArgument("slop", "2")
+    );
+    Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(
+        arguments, MatchPhraseQueryTest.this.matchPhraseQueryName)));
+  }
+
+  @Test
+  public void test_zero_terms_query_parameter_matchphrase_syntax() {
+    List<Expression> arguments = List.of(
+        dsl.namedArgument("field", "t1"),
+        dsl.namedArgument("query", "t2"),
+        dsl.namedArgument("zero_terms_query", "ALL")
+    );
+    Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(
+        arguments, MatchPhraseQueryTest.this.matchPhraseQueryName)));
+  }
+
+  @Test
+  public void test_zero_terms_query_parameter_lower_case_matchphrase_syntax() {
+    List<Expression> arguments = List.of(
+        dsl.namedArgument("field", "t1"),
+        dsl.namedArgument("query", "t2"),
+        dsl.namedArgument("zero_terms_query", "all")
+    );
+    Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(
```

Comment:
Conventionally, when writing an assert the expected value is the first argument. Second argument is the value to check.

Reply:
The `assertEquals` that was originally in `MatchPhraseIT.java` has been replaced by an `assertTrue` as suggested by another comment.